### DISCUSSION TITLE
Using Languages instead of Cultures

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#13192] [General]
Probably the sound is weird for those are using Localization APIs, but from users perspective most of the CMSs if not all are using the term languages instead, so the question do we really need to target specific culture?

I'd prefer using Arabic `ar` instead of `ar-Ye` or `ar-SA`, similarlly French `fr` instead of `fr-FR`, I'm not sure if I'm only the one who notice that, but while I worked with many CMSs last 10 - 15 years this was common

**Comment by jtkech** [comment]
First we need to complete the tweaks we are doing on cultures management ;) Then yes we could think about this kind of things, at an higher level I think, still having the cultures management behind the scene.
